test/blackbox-tests/test-cases/override-vendored_dirs/override-by-data_only_dirs-stanza.t/run.t

```diff
@@ -0,0 +1,5 @@
+The project "root_project" vendor "a" and "b" project.
+To override "a" project in "b", we use "data_only_dirs":
+"(subdir b (data_only_dirs a))"
```

Comment:
I found this a bit easier to understand, as it also explains that `b` vendors `a`:

```suggestion
The project "root_project" vendors projects "a" and "b", of which "b" vendors "a" itself.

To override the vendored "a" in project "b" `data_only_dirs` can be used: `(subdir b (data_only_dirs a))`
```